src/Extension/Propagator/B3/_register.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+\OpenTelemetry\SDK\FactoryRegistry::registerTextMapPropagatorFactory(
+    \OpenTelemetry\SDK\Common\Configuration\KnownValues::VALUE_B3,
+    \OpenTelemetry\Extension\Propagator\B3\B3SinglePropagatorFactory::class
```

Comment:
I would prefer if we add a `ClosureTextMapPropagatorFactory` (or allow registering closure factories directly) to avoid adding a factory class for every implementation.

Reply:
I've just removed the factories altogether, and registered the propagator instances instead. It's simpler, but also adding non-factories to a `FactoryRegistry` which is a little shady.

Reply:
Agree, that it's simpler, but as you written it breaks rule of storing only factories there. I don't have strong opinions for or against. Maybe, we should consider some renaming as `FactoryRegistry` will no longer store factories only.

Reply:
I've renamed it to `Registry`, which isn't very imaginative but seems valid.